packages/cubejs-api-gateway/src/sql-server.ts

```diff
@@ -332,7 +333,7 @@ export class SQLServer {
     // @todo Implement
   }
 
-  public async shutdown(): Promise<void> {
-    await shutdownInterface(this.sqlInterfaceInstance!);
+  public async shutdown(mode: ShutdownMode): Promise<void> {
```

Comment:
I don't think it does because there are only a few callers, and I don't have any reason to want a zero-argument `shutdown()` interface to be available.